swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+
+    # category-specific constructor using "Clothing" as the 
+    # default. 
+    def __init__(self, category="Clothing", condition=0):
```

Comment:
We should ideally remove category from the constructor's arguments. We don't want the caller to be able to set a different category. While it's possible that someone could change the category on an instance by making that assignment directly to it after instantiation, not passing that to the constructor at least indicates that they shouldn't do that.